src/lib/storage/disk_size.rb

```diff
@@ -204,7 +204,7 @@ def to_human_readable
         unit_index = 0
         size = @size_k.to_f
 
-        while size > 1024.0 && unit_index < UNITS.size - 1
+        while size >= 1024.0 && unit_index < UNITS.size - 1
```

Comment:
It's beautification: It gets you "1 TiB" instead of "1024 GiB" which is not exactly wrong, but I prefer the simpler way. BTW DiskSpace is one class that does have an extensive test suite, yet this was not caught. Yes, another test case could be added.